### PR TITLE
New Ideas

### PR DESCRIPTION
**Pull request by P0nt14c**
Adding 5 ideas.

Please accept

### CODE REVIEW

File: ideas/cool-project.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Talk about a Cool Project You Did
+
+|  Author  | Last Updated |       Time Estimate        |
+| :------: | :----------: | :------------------------: |
+| Jason Howe |  09/27/2022  | Short - 🟩🟩🟩⬜️⬜️ - Long |
+
+## Description
+
+Write a cool tool, and give a presentation on it!
```
Comment (BradHacker) [inline ideas/cool-project.md:9]: I like the shilling for the presentations at RITSEC, but this idea seems too generic to fit into the ideas list.

File: ideas/homelab.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Homelab
+
+|  Author  | Last Updated |       Time Estimate        |
+| :------: | :----------: | :------------------------: |
+| Jason Howe |  09/27/2022  | Short - 🟩🟩🟩🟩⬜️ - Long |
+
+## Description
+
+Do you have a homelab? Tell the club all about
```
Comment (BradHacker) [inline ideas/homelab.md:9]: I think this project would be more about deploying a homelab than presenting about it. These are project ideas more than presentation ideas.

**Review by BradHacker — CHANGES_REQUESTED** [review]
I like most of the ideas. Can you add these ideas to the existing lists and/or make a new list to categorize your ideas?

**Comment by chase1k** [comment]
Right now, I'm going to merge this. I might manually touch up the less elaborated ideas after. But for now, I am grateful for the contributions.

**Pull request merged**